README.md

```diff
@@ -1,155 +1,152 @@
-# OpenEOS for Unity
+# OpenEOS for Unity  
 [![openupm](https://img.shields.io/npm/v/com.robproductions.openeos?label=openupm&registry_uri=https://package.openupm.com)](https://openupm.com/packages/com.robproductions.openeos/) ![GitHub release (latest by date)](https://img.shields.io/github/v/release/RobProductions/OpenEOS?logo=github)
 
 An open-source port of the [Epic Online Services (EOS) SDK](https://dev.epicgames.com/docs/epic-online-services) in Unity Package format, with minor enhancements to provide a clean integration between EGS and Unity.
 
-<img width = "800" src="Documentation~/DocAssets/OpenEOSLogoBanner.jpg">
+<img width="800" src="Documentation~/DocAssets/OpenEOSLogoBanner.jpg">
 
 ## Overview
 
-OpenEOS offers 2 main services to your Unity project:
+OpenEOS offers two main services to your Unity project:  
 
-1. A port of the relevant EOS SDK files, including the C# codebase and DLLs configured to work within the Editor and builds. This allows you to import the EOS SDK as a package instead of having to manage the SDK on a project-by-project basis in your Assets folder.
-2. A few helper extensions and shortcuts for commonly used Epic Online Services features, such as SDK Initialization and Account Management.
+1. A port of the relevant EOS SDK files, including the C# codebase and DLLs configured to work within the Editor and builds. This allows you to import the EOS SDK as a package instead of managing the SDK on a project-by-project basis in your Assets folder.  
+2. A few helper extensions and shortcuts for commonly used Epic Online Services features, such as SDK initialization and account management.
 
 ### OpenEOS... Why?
 
-When researching the EOS SDK for use in my own projects, I was surprised to see only one or two usable plugins for integration with Unity (at the time of writing). Looking into those, they're pretty robust in their own right. However, they seem to offer full "manager" solutions that don't allow for much customization and essentially sit completely on top of the regular Online Services library.
+When researching the EOS SDK for use in my own projects, I was surprised to see only one or two usable plugins for integration with Unity (at the time of writing). Looking into those, they are robust in their own right. However, they offer full "manager" solutions that don't allow much customization and essentially sit on top of the regular Online Services library.
```

Comment:
Would prefer to leave the contraction and phrasing the way it is here so as not to imply certainty in this subject